packages/local_auth/local_auth_platform_interface/test/default_method_channel_platform_test.dart

```diff
@@ -0,0 +1,172 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:local_auth_platform_interface/default_method_channel_platform.dart';
+import 'package:local_auth_platform_interface/local_auth_platform_interface.dart';
+import 'package:local_auth_platform_interface/types/auth_options.dart';
+import 'package:local_auth_platform_interface/types/biometric_type.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+
+  const MethodChannel channel = MethodChannel(
+    'plugins.flutter.io/local_auth',
+  );
+
+  final List<MethodCall> log = <MethodCall>[];
+  late LocalAuthPlatform localAuthentication;
+
```

Comment:
Please add a test that DefaultLocalAuthPlatform is automatically registered as the instance.